nrf_security/Kconfig

```diff
@@ -750,7 +751,7 @@ choice
 	prompt "Backend selection"
 	depends on MBEDTLS_ECP_C && NRF_CRYPTO_BACKEND_COMBINATION_0
 	default CHOICE_CC310_MBEDTLS_ECP_C if CC310_BACKEND
-	default CHOICE_OBERON_MBEDTLS_ECP_C if !CC310_BACKEND
+	default CHOICE_OBERON_MBEDTLS_ECP_C if OBERON_BACKEND
```

Comment:
The pattern here is too eager. It will prioritize Oberon even when CC310 is enabled
I want this pattern to be OBERON_BACKEND && !CC310_BACKEND